.circleci/config.yml

```diff
@@ -59,18 +59,7 @@ jobs:
       - image: 'node:6'
         user: node
     steps: &unit_tests_steps
-      - checkout
```

Comment:
@nirupa-kumar Do you know why these changes are showing up? There are changes to .gitignore, the CircleCI config, common.cfg files, etc etc – all seemingly unrelated to the PR

Is it maybe just out-of-date and needs to have `master` merged into this branch?

I'd like to see these items as not part of this PR (which a merge/rebase might fix?)

Reply:
definitely were not added as part of product search - will do the merge/rebase fix.